src/styles/blocks/card.scss

```diff
@@ -0,0 +1,85 @@
+.card {
+  max-width: $card-max-width;
+  padding: 29px 14px 14px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    display: block;
+    margin: auto;
+    max-width: 160px;
+    min-height: 134px;
+    margin-bottom: 45px;
```

Comment:
margin-bottom is supposed to be less than 45px

Reply:
But on the mocup there is 45px gap from picture to title of the card. Why should it be less?

Reply:
you're right, my bad